libs/sdk-ui-dashboard/src/model/react/useDashboardEventDispatch.ts

```diff
@@ -0,0 +1,21 @@
+// (C) 2020-2021 GoodData Corporation
+import { useCallback } from "react";
+import { DashboardEvents } from "../events";
+import { useDashboardDispatch } from "./DashboardStoreProvider";
+
+/**
+ * Convenience hook for dispatching Dashboard events.
+ *
+ * @returns function that you can use to dispatch Dashboard events
+ * @alpha
+ */
+export const useDashboardEventDispatch = (): ((event: DashboardEvents) => void) => {
+    const dispatch = useDashboardDispatch();
+
+    return useCallback(
+        (event: DashboardEvents) => {
+            dispatch(event);
```

Comment:
Maybe I will add type guard to check event type in runtime since its public (prevent some dirty hacks)

`isDashboardEvent`

Reply:
added